src/sentry/migrations/0166_create_notificationsetting_table.py

```diff
@@ -1,9 +1,15 @@
 # Generated by Django 1.11.29 on 2021-02-26 22:11
 
 from django.db import migrations
+from enum import Enum
 import sentry.db.models.fields.bounded
 import sentry.models.integration
-import sentry.models.notificationsetting
+import sentry.notifications.types
```

Comment:
as I'm learning in my copy migration PR, you should copy this in instead of importing it

Reply:
I've pulled it out into https://github.com/getsentry/sentry/pull/24507.